pkg/apis/infra/v1alpha1/cluster.go

```diff
@@ -63,6 +63,9 @@ type ControlPlaneConfig struct {
 	// CertSANs sets extra Subject Alternative Names for the API Server signing cert.
 	// +optional
 	CertSANs []string `json:"certSANs,omitempty"`
+	// LoadBalancerDomainName is a variable used to set the endpoint for a Kubernetes cluster
+	// when a load balancer (LB) is enabled. The default value is "lb-apiserver.kubernetes.local".
+	LoadBalancerDomainName string `json:"loadBalancerDomainName,omitempty"`
```

Comment:
I think we should not dedault it to "lb-apiserver.kubernetes.local". 

It is common for user to have no domain, can we set it to a public address

Reply:
yes，we can do it